README.md

```diff
@@ -18,6 +18,17 @@ Press `?` to known about available keybindings in order to run actions on these
 
 You can also call the [API](#api) directly.
 
+## Configuration
+For Mac OS in .emacs configuration add following
+```
+(setenv "PATH" (concat (getenv "PATH") ":/usr/local/bin"))
+(setq exec-path (append exec-path '("/usr/local/bin")))
+;; Use "docker-machine env box" command to find out your environment variables
+(setenv "DOCKER_TLS_VERIFY" "1")
+(setenv "DOCKER_HOST" "tcp://10.211.55.3:2376")
+(setenv "DOCKER_CERT_PATH" "/Users/nexus/.docker/machine/machines/box")
+(setenv "DOCKER_MACHINE_NAME" "box")
+```
```

Comment:
The best way I found thanks to Silex is to use https://github.com/purcell/exec-path-from-shell

I end up with group customization done as on following screenshot.
https://www.dropbox.com/s/rd29jd8w2fast1q/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202016-01-05%2017.38.50.png?dl=0

But in case you have a lot of remote or swarm docker machines it’s more convenient to use your docker-machine code with exec-path-from-shell.

-- 
Максим Прокопов
http://nexusnotes.ru
https://www.facebook.com/mprokopov
skype me mprokopov

Включено 5 января 2016 г. в 17:30:29, Doug MacEachern (notifications@github.com) написал:

In README.md:

> @@ -18,6 +18,17 @@ Press `?` to known about available keybindings in order to run actions on these
>   
>  You can also call the [API](#api) directly.
>   
> +## Configuration
> +For Mac OS in .emacs configuration add following
> +```
> +(setenv "PATH" (concat (getenv "PATH") ":/usr/local/bin"))
> +(setq exec-path (append exec-path '("/usr/local/bin")))
> +;; Use "docker-machine env box" command to find out your environment variables
> +(setenv "DOCKER_TLS_VERIFY" "1")
> +(setenv "DOCKER_HOST" "tcp://10.211.55.3:2376")
> +(setenv "DOCKER_CERT_PATH" "/Users/nexus/.docker/machine/machines/box")
> +(setenv "DOCKER_MACHINE_NAME" "box")
> +```
Note that you can also use: docker-machine env --shell emacs

I'm currently using it like so: https://github.com/dougm/docker.el/blob/machine/docker-machine.el#L40

—
Reply to this email directly or view it on GitHub.